gallery/src/styles/NxList/NxListClickableLinksExample.tsx

```diff
@@ -11,8 +11,8 @@ import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 const NxListClickableLinksExample = () =>
   <ul className="nx-list">
     <li className="nx-list__item nx-list__item--clickable">
-      <a href="#/pages/nx-list" className="nx-list__link">
-        <span className="nx-list__text">nx-list page</span>
```

Comment:
> What do you think of changing the clickable link names from nx-table page to Table (HTML) page, nx-alert page to Alert (HTML) page etc...? I think it would make the documentation (doc) more clear in this case. But HTML variants are mentioned with the nx- format throughout the doc. It could be confusing to refer to an HTML variant by both its nx- format i.e. nx-table and its common name i.e. Table (HTML).
> 
> Thoughts?

While that's a valid question more broadly, in this case this code is within an example and the actual text or destinations of the links doesn't really matter.  You can change these ones or leave them, up to you.

Reply:
I'm going to leave my edits as is then 👍